Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -102,7 +102,9 @@ script:
       required: false
       secret: false
     deprecated: false
-    description: Sends a message to the specified teams.
+    description: |-
+      Sends a message to the specified teams.
+      If you want to mention a user please make sure to add ";" at the end of the mention. For example: @Bruce Willis;
```

Comment:
To mention a user in the message, add a semicolon ";" at the end of the user mention. For example: @JohnDoh;

Reply:
@kirbles19  We want them to know that a user can be assembled from one or more words, so it's better to give a full name for example. Is Bruce Willis a problem?